### DISCUSSION TITLE
Task displays as 'Running' without any progress

### DISCUSSION THREAD

**Discussion by homersimpsons** [BOINC/boinc#5894] [Q&A] [answered]
**Describe the bug**
Sometimes a task displays as "Running" but does not get any progress (no elapsed times counter, no percentage, no process in task manager).

**Steps To Reproduce**
1. Partiticipate to GPUGRID
2. Wait for an ATMML task
3. After the task as started for some times, pause it
4. Wait a bit and resume it
5. Here it should be stuck (but this does not always happen)

Suspending / resuming it again does not change anything. I have to stop the boinc manager (and the daemon) ans restart it so the task will restart directly.

**Expected behavior**
The task should start correctly

**Screenshots**
A screenshot of this task row, it was stuck like this for 30 minutes, other tasks were processing fine.
![Image](https://github.com/user-attachments/assets/95263772-3358-4d94-94dc-530748d575e3)

**System Information**
 - OS: Windows 11
 - BOINC Version: 8.0.2

**Additional context**
I reported this on GPUGRID a mont ago (https://www.gpugrid.net/forum_thread.php?id=5487).
Maybe priority applications on GPU has something to do with the issue

I tried to run a simulation with https://boinc.berkeley.edu/sim_web.php?action=show_scenario&name=212 but when I submit it I have an error page saying:
> command failed (139): ./sim --duration 600 --delta 60 --rec_half_life 864000 --infile_prefix scenarios/212/ --outfile_prefix scenarios/212/simulations/3/ 

Maybe I should also open an issue for this. The simulation seems to be created but empty though.

**Comment by AenBleidd** [comment]
This is the issue of the project application

**Reply by homersimpsons** [reply]
Okay, I will try to check this with GPUGRID then. About the issue with the simulation, should I raise it here?

**Comment by davidpanderson** [comment, marked as answer]
The simulator was broken; I fixed it, should work now
(although the simulator won't help with this issue).

I'm not sure this is a problem with the GPUGRID app.
If the client says the  job is running, its elapsed time should increase.
Vitalii, let's make this back into an issue and try to repro it.

**Reply by homersimpsons** [reply]
> The simulator was broken; I fixed it, should work now
> (although the simulator won't help with this issue).

I confirm the emulator is fixed, but in fact I do not think it reproduces the issue.

> I'm not sure this is a problem with the GPUGRID app.
> If the client says the job is running, its elapsed time should increase.
> Vitalii, let's make this back into an issue and try to repro it.

Thanks for re-opening, hope you can find the issue, for the record it was re-opened as https://github.com/BOINC/boinc/issues/5895
